src/openrct2/object/BannerObject.cpp

```diff
@@ -26,7 +26,7 @@ void BannerObject::ReadLegacy(IReadObjectContext* context, OpenRCT2::IStream* st
     _legacyType.banner.scenery_tab_id = OBJECT_ENTRY_INDEX_NULL;
     stream->Seek(2, OpenRCT2::STREAM_SEEK_CURRENT);
 
-    GetStringTable().Read(context, stream, OBJ_STRING_ID_NAME);
```

Comment:
Modify string table read to take the new type. Modify the set function as well.

Reply:
There are following functions in StringTable.h, class StringTable:
```
void Read(IReadObjectContext* context, OpenRCT2::IStream* stream, uint8_t id);
std::string GetString(uint8_t id) const;
std::string GetString(uint8_t language, uint8_t id) const;
void SetString(uint8_t id, uint8_t language, const std::string& text);
```
and as I undertand field `uint8_t id` is the field for `ObjectStringID`.
I want to replace all `uint8_t id` fields with `ObjectStringID id`. Is that ok, or should I pay more attention here for something else?

Reply:
I've also modified `Object::GetString` function in Object.cpp and modified `auto sz = GetOverrideString(index);` to `auto sz = GetOverrideString(static_cast<uint8_t>(index));`